lib/features/manual/widget/showcase_text_style.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class ShowcaseTextStyle extends StatelessWidget {
+  const ShowcaseTextStyle({
+    required this.descriptionText,
+    super.key,
+  });
+
+  final String descriptionText;
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(
+      descriptionText,
+      style: const TextStyle(
```

Comment:
Should be one of the existing text styles from the theme, but that can happen in the next PR.